test/AlgebraicGeometry/Schemes/ProjectiveSchemes.jl

```diff
@@ -312,3 +312,36 @@ end
   @test issubset(X1, P2)
   @test issubset(X1, X2)
 end
+
+@testset "closed embeddings" begin
+  IP2 = projective_space(QQ, 2)
+  S = homogeneous_coordinate_ring(IP2)
+  (x, y, z) = gens(S)
+  I = ideal(S, [x^2 + y^2 + z^2])
+  X, inc = sub(IP2, I)
+  X, inc = sub(IP2, gens(I))
+
+  J = ideal(S, [x+y+z])
+  X2, inc2 = sub(IP2, J)
```

Comment:
Line 321 is unnecessary and overwritten by line 322. The signature of line 321 is also tested in line 325

Reply:
Here the test for the signature is more explicit. So I would say it is okay as is.

Reply:
fine with me